Stripe/STPBankSelectionViewController.m

```diff
@@ -76,6 +94,12 @@ - (BOOL)useSystemBackButton {
     return YES;
 }
 
+- (void)updateWithBankStatus:(STPFPXBankStatusResponse *)bankStatusResponse {
+    self.bankStatus = bankStatusResponse;
+    
+    [self.tableView reloadData];
```

Comment:
Any reason _not_ to do a `begin/EndUpdates` here rather than a `reloadData`?

Reply:
We're reloading all cells, but I suppose we could.